src/main/java/com/audiolibrary/web/service/ArtistService.java

```diff
@@ -0,0 +1,155 @@
+package com.audiolibrary.web.service;
+
+import com.audiolibrary.web.model.Album;
+import com.audiolibrary.web.model.Artist;
+import com.audiolibrary.web.repository.AlbumRepository;
+import com.audiolibrary.web.repository.ArtistRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import javax.persistence.EntityNotFoundException;
+import java.security.Signature;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ArtistService {
+
+    @Autowired
+    private ArtistRepository artistRepository;
+
+    @Autowired
+    private AlbumRepository albumRepository;
+
+    @Value("${deleteAlbumsWithArtist}")
+    private Boolean DELETE_ALBUMS_WITH_ARTIST;
```

Comment:
Utilise cette syntaxe uniquement pour les constantes
```suggestion
    private Boolean deleteAlbumsWithArtist;
```